coding-the-legislation/legislation_parameters.md

```diff
@@ -1,93 +1,147 @@
 # Legislation parameters
 
-## Definition
 
-The legislation parameters are stored in XML files in the directory `parameters` ([example in OpenFisca-France](https://github.com/openfisca/openfisca-france/blob/master/openfisca_france/parameters)).
+## Hierarchical structure
```

Comment:
I had a lot of trouble understanding part.
We can work on it tomorrow if it can help :)